examples/standalone-airgap-dev/README.md

```diff
@@ -1,26 +1,66 @@
-# EXAMPLE: Standalone, Airgapped Package Installation of Terraform Enterprise (bootstrapping with airgap prerequisites)
+# EXAMPLE: Standalone External Airgapped Installation of Terraform Enterprise
 
-## About this example
+## About This Example
 
-This example for Terraform Enterprise creates a TFE
-installation with the following traits.
+This example for Terraform Enterprise creates a TFE installation with the following traits:
 
-- Standalone
-- _Mocked_ Airgapped installation
-- External mode
-- a small VM machine type (m5.xlarge)
-- Ubuntu 20.04 as the VM image
-- a publicly accessible HTTP load balancer with TLS termination
-- no proxy server
+-  Standalone
+-  External Services production type
+-  Airgapped (mocked)
+-  This example merely tests that the `airgap_url` package is able to install TFE. It does not, however, assume that the environment is air-gapped, and it therefore installs the prerequisites for an air-gapped installation too. 
+-  m5.xlarge virtual machine type
+-  Ubuntu 20.04
+-  A publicly accessible HTTP load balancer with TLS termination
```

Comment:
```suggestion
- Standalone
- External Services production type
- Air gapped (mocked)
- This example merely tests that the `airgap_url` package is able to install TFE. It does not, however, assume that the environment is air gapped, and it therefore installs the prerequisites for an air gapped installation, too. 
- m5.xlarge virtual machine type
- Ubuntu 20.04
- A publicly accessible HTTP load balancer with TLS termination
```

You somehow ended up with extra spaces. Also, I guess we should standardize on the proper spelling of air gapped as two separate words (in anything other than the existing variables).